cmd/repo.go

```diff
@@ -284,27 +297,18 @@ func (index *RepoIndex) listProjects(repoName string) (string, error) {
 		Debug.log("Adding unsupported repoistory", repoName)
 		unsupportedRepos = append(unsupportedRepos, repoName)
 	}
-	//table.AddRow("REPO", "ID", "VERSION", "TEMPLATES", "DESCRIPTION")
-	table.AddRow("REPO", "ID", "VERSION ", "DESCRIPTION")
+	table.AddRow("REPO", "ID", "VERSION  ", "TEMPLATES", "DESCRIPTION")
 
 	Stacks, err := index.buildStacksFromIndex(repoName, Stacks)
 	if err != nil {
 		return "", err
 	}
 
 	for _, value := range Stacks {
-		table.AddRow(value.repoName, value.ID, value.Version, truncate(value.Description, 80))
-
+		table.AddRow(value.repoName, value.ID, value.Version, value.Templates, value.Description)
```

Comment:
Why was `truncate` removed from description?

Reply:
was removed because wrap is a more elegant solution.